docs/src/index.md

```diff
@@ -1,509 +1,4 @@
-
-Currently there are two dense predictor types, ```DensePredQR``` and
-```DensePredChol```, and the usual caveats apply.  The Cholesky
-version is faster but somewhat less accurate than that QR version.
-The skeleton of a distributed predictor type is in the code
-but not yet fully fleshed out.  Because Julia by default uses
-OpenBLAS, which is already multi-threaded on multicore machines, there
-may not be much advantage in using distributed predictor types.
-
-A ```ModResp``` type must provide methods for the ```wtres``` and
-```sqrtxwts``` generics.  Their values are the arguments to the
-```updatebeta``` methods of the ```LinPred``` types.  The
-```Float64``` value returned by ```updatedelta``` is the value of the
-convergence criterion.
-
-Similarly, ```LinPred``` types must provide a method for the
-```linpred``` generic.  In general ```linpred``` takes an instance of
-a ```LinPred``` type and a step factor.  Methods that take only an instance
-of a ```LinPred``` type use a default step factor of 1.  The value of
-```linpred``` is the argument to the ```updatemu``` method for
-```ModResp``` types.  The ```updatemu``` method returns the updated
-deviance.
-
-## API
-
-### Types defined in the package
-
-```@docs
-DensePredChol
-DensePredQR
-GlmResp
-LinearModel
-LmResp
-LinPred
-GLM.ModResp
-```
-
-### Constructors for models
-
-The most general approach to fitting a model is with the `fit` function, as in
-```jldoctest
-julia> using Random;
-
-julia> fit(LinearModel, hcat(ones(10), 1:10), randn(MersenneTwister(12321), 10))
-LinearModel{LmResp{Array{Float64,1}},DensePredChol{Float64,LinearAlgebra.Cholesky{Float64,Array{Float64,2}}}}:
-
-Coefficients:
-      Estimate Std.Error  t value Pr(>|t|)
-x1    0.717436  0.775175 0.925515   0.3818
-x2   -0.152062  0.124931 -1.21717   0.2582
-```
-
-This model can also be fit as
-```jldoctest
-julia> using Random;
-
-julia> lm(hcat(ones(10), 1:10), randn(MersenneTwister(12321), 10))
-LinearModel{LmResp{Array{Float64,1}},DensePredChol{Float64,LinearAlgebra.Cholesky{Float64,Array{Float64,2}}}}:
-
-Coefficients:
-      Estimate Std.Error  t value Pr(>|t|)
-x1    0.717436  0.775175 0.925515   0.3818
-x2   -0.152062  0.124931 -1.21717   0.2582
-```
-
-### Model methods
-```@docs
-GLM.cancancel
-delbeta!
-StatsBase.deviance
-GLM.dispersion
-GLM.installbeta!
-GLM.issubmodel
-linpred!
-linpred
-lm
-StatsBase.nobs
-StatsBase.nulldeviance
-StatsBase.predict
-updateμ!
-wrkresp
-GLM.wrkresp!
-```
-<!-- ftest should be included as well but currently Documenter can't parse the doctest for ftest-->
-
-### Links and methods applied to them
-```@docs
-Link
-GLM.Link01
-CauchitLink
-CloglogLink
-IdentityLink
-InverseLink
-InverseSquareLink
-LogitLink
-LogLink
-NegativeBinomialLink
-ProbitLink
-SqrtLink
-linkfun
-linkinv
-mueta
-inverselink
-canonicallink
-glmvar
-mustart
-devresid
-GLM.dispersion_parameter
-GLM.loglik_obs
-```
+## Package summary
+Linear and generalized linear models in Julia
```

Comment:
The home page looks quite weird now since it's empty. How about putting the "Manual" section in the home page?